.github/workflows/release-bot.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - 'release/*'
 
 permissions:
-  contents: read
```

Comment:
I don't think this should be required given that we do not call any workflows that would need to elevate permissions